lib/tag_editor.dart

```diff
@@ -149,14 +146,14 @@ class _TagsEditorState extends State<TagEditor> {
   }
 
   void _onSubmitted(String string) {
-    widget.onSubmitted(string);
+    widget.onSubmitted!(string);
```

Comment:
Maybe can do optional call instead of force unwrapping?
```suggestion
    widget.onSubmitted?.call(string);
```